apps/ledger-live-desktop/src/renderer/screens/account/index.jsx

```diff
@@ -90,6 +92,8 @@ const AccountPage = ({
     ? perFamilyAccountSubHeader[mainAccount.currency.family]
     : null;
   const bgColor = useTheme("colors.palette.background.paper");
+  const ethStakingProviders = useFeature("ethStakingProviders");
```

Comment:
No we can't because it's a hook - and that ETH logic is in a regular function. This is an issue with the way we are passing the hooks around to the currency specific files. I was going to raise this with you actually, we can refactor refactor ¦getBannerProps¦ to `useGetBannerProps`, and then each staking program can have a hook, i.e. `useGetCosmosBannerProps` etc. 

Let me know what you think?